### ISSUE TITLE
[BUG] H012 false positive in JS

### ISSUE THREAD

**Issue by jayvdb** [Riverside-Healthcare/djLint#40]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: Windows
 - Python Version: 3.9
 - djLint Version: 0.4.9

## Issue
`<=` in JS is causing H012 

## How To Reproduce
```html
<script>
        $(document).ready(function () {
            // Parts
            for (i = 1; i <= 3; i++) {
            }

    </script>
```

**Comment by christopherpickering** [comment]
Also related to #41 , we should find no errors in a script, or other ignored blocks.
